src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -52,6 +56,16 @@ class SidebarScreen extends Component {
     componentDidMount() {
         Performance.markStart(CONST.TIMING.SIDEBAR_LOADED);
         Timing.start(CONST.TIMING.SIDEBAR_LOADED, true);
+
+        if (this.props.isFirstTimeNewExpensifyUser) {
+            // For some reason, the menu doesn't open without the timeout
+            setTimeout(() => {
+                this.toggleCreateMenu();
+
+                // Set the NVP back to false (this may need to be moved if this NVP is used for anything else later)
```

Comment:
>(this may need to be moved if this NVP is used for anything else later)

This comment isn't really useful. Why are we thinking about what we _might_ do with this _later_. Let's just focus on what we are using it for now because we don't know how else it would be used or why that would make a difference to the code here.